core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
I hear you. I guess in my mind anything that's in the config *should* be meant to be configurable / changed in some case that warrants it. I feel like we could make a case that this statement is true for ~~almost~~ everything currently in our conf ? Maybe it isn't in which case yes there's no reason to make an exception. My main argument is that I really don't see *any* reason why someone would want to change that - or even be aware that it exists.